libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager+Internal.h

```diff
@@ -27,7 +27,6 @@
 @interface SalesforceSDKManager () <SalesforceSDKManagerFlow, SFUserAccountManagerDelegate>
 {
     BOOL _isLaunching;
-    UIViewController* _defaultSnapshotViewController;
```

Comment:
I think the premise is, regardless of custom or default, there will only ever be one at a time, so why not just have one variable for it.